Sunrise/ViewModels/ProductViewModel.swift

```diff
@@ -16,16 +18,40 @@ class ProductViewModel {
     let price = MutableProperty("")
     let oldPrice = MutableProperty("")
     let imageUrl = MutableProperty("")
+    let quantities = (1...9).map { String($0) }
+    let isLoading = MutableProperty(false)
+
+    // Dialogue texts
+    let addToCartSuccessTitle = NSLocalizedString("Product added to cart", comment: "Product added to cart")
+    let addToCartSuccessMessage = NSLocalizedString("You have successfully added product to your cart. Would you like to continue looking for more, or go to cart overview?", comment: "Product added to cart message")
```

Comment:
I think we can drop the first sentence, as the title already contains that info?